isv.md

```diff
@@ -65,6 +65,7 @@ Additionally:
 | Observability | [Sumo Logic](https://partners.amazonaws.com/partners/001E000000Rl0y6IAB) | [Collector (OpenTelemetry)](https://help.sumologic.com/Beta/Sumo_Logic_Distribution_for_OpenTelemetry) | Release: Beta: https://github.com/SumoLogic/sumologic-otel-collector/blob/main/docs/Installation.md#linux-on-arm64<br>Announcement: (https://support.sumologic.com/hc/en-us/articles/360034233293-Is-ARM64-architecture-supported-by-Sumo-Logic-collector-installer-) |
 | Security | [Aqua Security](https://partners.amazonaws.com/partners/001E000001LiLQqIAN) | [Aqua Cloud Native Security Platform](https://www.aquasec.com/aqua-cloud-native-security-platform/) | Release: https://www.aquasec.com/demo/<br>Announcement: https://blog.aquasec.com/aqua-support-aws-graviton |
 | Security | [Aqua Security](https://partners.amazonaws.com/partners/001E000001LiLQqIAN) | [Open Source (Trivy, kube-bench)](https://www.aquasec.com/products/open-source-projects/) | Release: https://gallery.ecr.aws/?architectures=ARM+64&verified=verified&searchTerm=aqua<br>Announcement: https://github.com/aquasecurity/trivy/releases/, https://github.com/aquasecurity/kube-bench/releases |
+| Security | [Bloombase](https://partners.amazonaws.com/partners/001E000000gLbOIIA0) | [Bloombase StoreSafe](https://www.bloombase.com/products/storesafe) | Release: https://supportal.bloombase.com<br>Announcement: Graviton2 listed since Bloombase StoreSafe 3.4.8: https://supportal.bloombase.com |
```

Comment:
The Hyperlink for Bloombase to the partner page is broken, can you fix it please?

Do you have any better direct link to the announcement of Graviton compatibility.  Best I could find is this Youtube video, but it is directly from Bloombase: https://www.youtube.com/watch?v=WsOoR0zTgho